### ISSUE TITLE
Window position is weird...

### ISSUE THREAD

**Issue by NaturalmotionStudios** [GridTech123/Company-Sim-Early-Access#1]
When I start the application, it opens at twice the resolution of my PC.

**Comment by GridTech123** [comment]
yes, pygame does not resize objects. in settings there is a resizable option but it doesn't resize objects on the screen, just the window.

**Comment by NaturalmotionStudios** [comment]
Yeah, but could you make the game's resolution smaller, maybe about 1920 x 1080?

**Comment by GridTech123** [comment]
the game runs at 1800 x 1000

**Comment by NaturalmotionStudios** [comment]
Not for me it doesn't...

**Comment by GridTech123** [comment]
weird here is the game code telling it that

`pygame.init()
screen_x = 1800
screen_y = 1000
screen = pygame.display.set_mode([screen_x,screen_y])`

**Comment by GridTech123** [comment]
I am now planning on redesigning the UI for smaller screens, I will also make download times faster

**Comment by NaturalmotionStudios** [comment]
Okay, I'll re-update the installer for it sometime today.

**Comment by NaturalmotionStudios** [comment]
Okay, I'll update the installer when you release it.
